libs/barista-components/expandable-section/src/expandable-section.spec.ts

```diff
@@ -249,6 +249,27 @@ describe('DtExpandableSection', () => {
       changedSubscription.unsubscribe();
     });
   });
+
+  fdescribe('dt-expandable-section static attributes', () => {
```

Comment:
please remove the f as it only executes this test

```suggestion
  describe('dt-expandable-section static attributes', () => {
```